apps/connect/source/ftrack_connect/ui/application.py

```diff
@@ -29,13 +29,17 @@
 import ftrack_connect
 import ftrack_connect.event_hub_thread as _event_hub_thread
 import ftrack_connect.error
-from ftrack_connect.util import (
+from ftrack_connect.utils.plugin import (
     get_default_plugin_directory,
     get_plugins_from_path,
     get_plugin_data,
-    open_directory,
     PLUGIN_DIRECTORIES,
 )
+from ftrack_connect.utils.login import (
+    read_json_config,
+    write_json_config,
+)
+from ftrack_connect.utils.directory import open_directory
 import ftrack_connect.ui.theme
```

Comment:
Perhaps change all these to the form `from ftrack_connect.ui import theme` and then use `theme.` to align with rest of code base.

Reply:
I think we should keep that for another task, we will never end if we try to clean the connect code to much.